gradle-baseline-java/build.gradle

```diff
@@ -15,6 +15,7 @@ dependencies {
     compile 'net.ltgt.gradle:gradle-errorprone-plugin'
     compile 'org.apache.maven.shared:maven-dependency-analyzer'
     compile 'org.github.ngbinh.scalastyle:gradle-scalastyle-plugin_2.11'
+    compile 'com.google.googlejavaformat:google-java-format'
```

Comment:
this is kinda interesting... I'd prefer to actually put this in a configuration, and resolve it only when the `format` task is called, as this would reduce the amount of stuff necessary for configuration.  HOWEVER, I think that would require using a JavaExec task, which I don't want to do.

Reply:
Actually it won't, Spotless allows you to load a classpath and run it in a different classloader. We could do that instead